.github/workflows/snyk.yml

```diff
@@ -0,0 +1,19 @@
+name: Snyk CI check
```

Comment:
Do you mind adding a comment about how often this job will run?

Reply:
Good suggestion, thanks. Done.